Regression/Checksum/benchmarks_json/RefinedInjection.json

```diff
@@ -2,45 +2,45 @@
   "beam": {
     "particle_cpu": 0.0,
     "particle_id": 5050.0,
-    "particle_momentum_x": 4.561019131383119e-20,
-    "particle_momentum_y": 4.149842919134552e-20,
-    "particle_momentum_z": 1.364708476750903e-17,
-    "particle_position_x": 4.652770082118157e-05,
+    "particle_momentum_x": 4.5610191313831193e-20,
+    "particle_momentum_y": 4.1498429191345516e-20,
+    "particle_momentum_z": 1.3647084767509034e-17,
+    "particle_position_x": 4.6527700821181575e-05,
     "particle_position_y": 0.001484501913502292,
     "particle_weight": 12483018148921.525
   },
   "electrons": {
-    "particle_cpu": 15846.0,
-    "particle_id": 136508842.0,
-    "particle_momentum_x": 1.6448471633168762e-19,
-    "particle_momentum_y": 7.798609849233592e-19,
-    "particle_momentum_z": 3.2439317401014397e-19,
-    "particle_position_x": 0.13260655641686475,
-    "particle_position_y": 0.28106090693937097,
-    "particle_weight": 394602539062500.8
+    "particle_cpu": 4788.0,
+    "particle_id": 13694226.0,
```

Comment:
In case it crashes again, did you run with `warpx.refine_plasma=1`?
https://github.com/ECP-WarpX/WarpX/blob/0c88663f7e38daa3770bb2888480e8747bfd4aed/Regression/WarpX-tests.ini#L1135